backend/src/repositories/CompanyRepository.ts

```diff
@@ -0,0 +1,94 @@
+import { v4 as uuidv4 } from "uuid";
+import { GoogleSpreadsheetRow } from "google-spreadsheet";
+import { getDatabaseInstance } from "../database";
+
+import { Company } from "@/models/Company";
+import { DATABASE_CONSTANTS } from "@/constants/databaseConstants";
+import { AddCompanyRequestDto } from "@/models/dtos/AddCompanyRequestDto";
+
+
+async function createCompany(companyInfo: AddCompanyRequestDto) {
```

Comment:
for the add/update/get API endpoints we should bubble the error from the backend to the frontend console so that the user has some idea about the exact issue. Right now we are swallowing the error and displaying a fixed error message in the client for all instances.